### DISCUSSION TITLE
Question: How does the generation via script work?

### DISCUSSION THREAD

**Discussion by Vortilion** [sumpfork/dominiontabs#425] [Q&A] [answered]
I checked out the branch via github and tried running it locally. I did pip install domdiv, but if I try to use the dominion_dividers command, it says it doesn't know it.

**Comment by md5i** [comment]
Did you install it?
`python3 setup.py install --local`

**Comment by sumpfork** [comment, marked as answer]
So technically, there are two different things here:
1) You can do a `pip install domdiv` without checking out anything from github because I publish the `domdiv` package to the official Python PyPi index. If your python and pip set up correctly, your shell should find the `dominion_dividers` command after you do that.
2) If you want to do development on the code, you should do a github checkout. Then, as @md5i says, doing a local install or a `python setup.py develop` or a `pip install -e .` (both of which install the package in a way that points to the checked out source so you can see changes at you edit the code) are the right thing.

**Reply by sumpfork** [reply]
Note that 2. is out of date as of #506. The right thing is now `pip install -e .`, there is no `setup.py` anymore. You probably also want to `pip install -r dev-requirements.txt` first.

**Reply by nickv2002** [reply]
You can also use docker to run the script now. [See examples in the README.md](https://github.com/sumpfork/dominiontabs/blob/master/README.md#docker)
